src/Mod/TechDraw/Gui/QGVPage.cpp

```diff
@@ -308,15 +307,10 @@ void QGVPage::deactivateHandler()
 
 void QGVPage::startBalloonPlacing(DrawView* parent)
 {
-    //    Base::Console().Message("QGVP::startBalloonPlacing(%s)\n", parent->getNameInDocument());
```

Comment:
Personally i like removing debug code like this, but I don't know if that is a shared view. Maybe it shouldn't be sneaked in as a change in this pr :)

Reply:
@wandererfan has begun removing them since the linters were introduced. So I think it's fine